test/user.test.js

```diff
@@ -1763,6 +1763,367 @@ describe('User', function() {
     });
   });
 
+  describe('Email Update', function() {
+    describe('User changing email property', function() {
+      var usersId;
+      var currentEmailCredentials = { email: 'm@c.com', password: 'bar' };
+      var updatedEmailCredentials = { email: 'n@d.com', password: 'bar' };
+
+      beforeEach(function(done) {
+        async.series([
+          function(next) {
+            User.create({ email: currentEmailCredentials.email,
+            password: currentEmailCredentials.password }, function(err, user) {
+              usersId = user.id;
+              next();
+            });
+          },
+          function(next) {
+            User.login({ email: currentEmailCredentials.email,
+            password: currentEmailCredentials.password }, function(err, accessToken1) {
+              if (err) return next(err);
+              assert(accessToken1.userId);
+              next();
+            });
+          },
+          function(next) {
+            User.login({ email: currentEmailCredentials.email,
+            password: currentEmailCredentials.password }, function(err, accessToken2) {
+              if (err) return next(err);
+              assert(accessToken2.userId);
+              next();
+            });
+          },
+        ], function(err) {
+          if (err) return done(err);
+          done();
+        });
+      });
+
+      afterEach(function(done) {
+        User.deleteById(usersId, function(err, user) {
+          if (err) return done (err);
+          done();
+        });
+      });
+
+      it('invalidates sessions once the email is changed using `updateAttribute`', function(done) {
+        async.series([
+          function(next) {
+            User.findById(usersId, function(err, userFound)  {
+              if (err) return next(err);
+              userFound.updateAttribute('email', updatedEmailCredentials.email,
+            function(err, userInstance) {
+              if (err) return next(err);
+              assert.equal(userInstance.email, updatedEmailCredentials.email);
+              next();
+            });
+            });
+          },
+          function(next) {
+            AccessToken.find({ where: { userId: usersId }}, function(err, tokens) {
+              if (err) return next(err);
+              expect(tokens.length).to.equal(0);
+              next();
+            });
```

Comment:
This function is shared by many tests, please refactor and extract a shared function.

```js
function assertNoAccessTokens(cb) {
  AccessToken.find(...);
}
```